File: source codes/Lab 6 - Sasha Angermueller/PrintSum.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    int countingnum;
    int x = 1;
    Console.Write("You entered: ");
    countingnum = int.Parse(Console.ReadLine());
    int sum = 0;
    if (countingnum < 0 || countingnum > 100)
      {Console.WriteLine("Invalid input. Try again.");}
    else {
    while (x <= countingnum) 
      {sum += x;x++;}
    Console.WriteLine("Sum of values: " + sum);}
  }
}

File: School source codes/Lab 6/PrintSum.rb
print "You entered: "
countingnum = gets.chomp.to_i
sum = 0
x = 1
if (countingnum < 0 || countingnum > 100)
  puts "Invalid input. Try again."
else 
  while (x <= countingnum) 
    sum += x
    x += 1
  end
end
puts "Sum of values: #{sum}"